### DISCUSSION TITLE
FN4 MyOpenMath

### DISCUSSION THREAD

**Discussion by kathypinzon** [TeamBasedInquiryLearning/precalculus#369] [General]
Reviewer: @AbbyANoble 

1-9 write equation given description of transformations
10-14 describe effects of transformations on graph
15-18 write equation from graph
19, 20 write equation from table and points
21-26 graph transformed functions
27 describe domain of transformed functions (not sure if this was necessary or not but I know we discuss it in exponential and logs later).

I didn't include any examples with exponentials and logs since we did transformations of those in the later section.

**Comment by AbbyANoble** [comment]
Questions 3 and 5 are the same, except 5 includes a video to help.

**Comment by AbbyANoble** [comment]
Question 6 says "stretched by _" on each version, but doesn't indicate vertical or horizontal. The linked video specifies vertical in the one example it does.

**Reply by AbbyANoble** [reply]
Looks like this happens in questions 6-9.

**Comment by AbbyANoble** [comment]
I felt like there weren't many with a reflection in the homework.
